### PR TITLE
Red Hat article delivery

### PR DESCRIPTION
**Pull request by Abdou-kun**
Here is the Red Hat article

### CODE REVIEW

**Review by cr105ph1nx — CHANGES_REQUESTED** [review]
Hello Abdou, can you please add the titles of the articles you added to the main README.md? with a link that redirects to them. Don't mind the merge conflict I will resolve it.

**Comment by abdelhak2406** [comment]
@cr105ph1nx don't forget to  squash it before merge

**Comment by cr105ph1nx** [comment]
OMC maintainers are faster than speed sahit jeune

**Pull request merged**
